scripts/std-lib-regression.sh

```diff
@@ -9,9 +9,13 @@ PLATFORM=$(uname -sp)
 if [[ $PLATFORM == "Linux x86_64" ]]
 then
   TARGET="x86_64-unknown-linux-gnu"
+  # 'env' necessary to avoid bash built-in 'time'
+  WRAPPER="env time -v"
 elif [[ $PLATFORM == "Darwin i386" ]]
 then
   TARGET="x86_64-apple-darwin"
+  # mac 'time' doesn't have -v
+  WRAPPER=""
```

Comment:
My thought was just: this change was just to make it easier to go to CI and see the number there if we want to know it. I don't expect it to vary much by platform.

I can see how it might be nice to use locally. How about instead of installing gnu-utils, I just test if `gtime` is available and use it if it is?